### DISCUSSION TITLE
修改部分文件和容器失败

### DISCUSSION THREAD

**Discussion by kohirei** [1Panel-dev/1Panel#5932] [Q&A]
### 1Panel 版本

v1.10.13

### 问题描述

在面板里编辑文件和部分容器时候会触发bug，返回到主页文件和容器没修改成功
面板日志没有报错，不知道什么问题

### 重现步骤

.

https://github.com/user-attachments/assets/ae7391b5-1cb8-4708-a7a2-0b5575f6d461

**Comment by wanghe-fit2cloud** [comment]
是通过反向代理方式访问的 1Panel 面板吗？
> 可以先排查下是不是被 WAF 拦截导致的。

**Reply by kohirei** [reply]
是waf的问题解决了
